browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -69,8 +69,13 @@ inline constexpr const char* kAllowedUrlPrefixes[] = {
     "https://p3a.brave.com/",
     "https://star-randsrv.bsg.brave.com/",
 
+    // ads
+    "https://geo.ads.bravesoftware.com",
+    "https://static.ads.bravesoftware.com",
+
     // Other
     "https://brave-core-ext.s3.brave.com/",
+    "https://cdn.search.brave.com/",
```

Comment:
why are we connecting to the search CDN before search is loaded? is this from the NTP widget?

Reply:
I think so. Test catched this url since `1.68.30` and https://github.com/brave/brave-core/commit/bd1c2a2c2216eed3a67a7dd984a43c232c1ebe71 is added at that time.

Reply:
What are we actually pulling from the CDN? The logo?

Reply:
Only favicon - https://cdn.search.brave.com/serp/favicon.ico

Reply:
can we just bundle it?

Reply:
@diracdeltas Sorry, can't understand. what bundle means?

Reply:
i mean include it in the browser so it doesn't need to be loaded remotely

Reply:
Ah, got it. I think this favicon request comes from the brave logo inside the search widget.
<img width="578" alt="image" src="https://github.com/brave/brave-core/assets/6786187/33c19b7b-eb7b-4dc3-b7e0-170a4a213a25">
If we set brave search as a default always, I think we could use bundled from there. @fallaciousreasoning WDYT?

Reply:
The user can pick their default (it'll remember the last search engine they used). Do we bundle icons for all the search engines?

Reply:
i guess this is fine since hopefully it will be cached. @fmarier WDYT?

Reply:
Since this is not a high-priority problem, let's merge this PR to resolve the test failure and then discuss in the new follow-up issue I just created: https://github.com/brave/brave-browser/issues/38503